taxonomies/labels.txt

```diff
@@ -19712,3 +19712,11 @@ fr:Sans résidus de pesticides, sans résidu de pesticide, zéro résidus de pes
 # obligation of means: cultivated without pesticides (but can be a specific year, or after flowers bloom)
 en:Pesticide-free, without pesticide, no pesticide
 fr:Sans pesticides, cultivé sans pesticides
+
+fr:Prix Juste Producteur
```

Comment:
@teolemon and @aleene do you think it should be the child of some category ?

Maybe we could have a common parent with fair trade ?

Reply:
We still need to work on that. I need to restart that work. Add a comment above it, specifying this is a fairtrade-label. And put it under Max Havelaar somewhere, so that we start to group those labels.

Reply:
Hi @alexgarel and @aleene, do you want to resolve fairtrade-label in this pull request or do it later ? Just to know if we can merge this. I didn't hear anything from the label authority ...

Reply:
we will resolve later :-)